File: rest/call/list-get-example-3/list-get-example-3.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# start_time param must be of Date type
@start_time = Time.new('2009-07-06')

# Get all calls for given account which have the status of completed and given
# start time and print the call recepient
@client.account.calls.list(status: 'completed', start_time: @start_time)
       .each do |call|
         puts call.to
       end

File: rest/incoming-phone-numbers/list-get-example-3/list-get-example-3.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;
using Twilio.Types;

class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/console
    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    const string authToken = "your_auth_token";
    TwilioClient.Init(accountSid, authToken);

    var phoneNumber = new PhoneNumber("867");
    var numbers = IncomingPhoneNumberResource.Read(phoneNumber: phoneNumber);

    foreach (var number in numbers)
    {
      Console.WriteLine(number.VoiceUrl);
    }
  }
}
